includes/compatibility/class-cpt-compat-mode.php

```diff
@@ -174,4 +175,18 @@ public function enfold_title( $args, $id ) {
 		}
 		return $args;
 	}
+
+	/**
+	 * Display the BD-plugin title if Enfold header is disabled.
+	 *
+	 * @since x.x
+	 */
+	public function enfold_maybe_add_single_title() {
+		/** @phpstan-ignore-next-line */
```

Comment:
Why does this need to be ignored?

A lot of this looks like duplication. Is it possible to merge this duplicate code with the theme_enfold_maybe_add_single_title function?

Reply:
Yes, of course. I saw that some code was repeated to make it compatible, and I decided to act accordingly so as not to mess up the order.

Reply:
I'm not seeing what the error was, but I'm guessing it was undefined function? If so, we need to add avia_header_setting() into the stubs.php in order to remove this ignore statement.

Reply:
I removed it in this [commit](https://github.com/Strategy11/business-directory-plugin/pull/280/commits/974b4a485676b15d2b7e7267f7e907573d9614ec).

Reply:
It's still here. It looks like it got removed from one function but not the other.

Reply:
Sorry, I just removed the other one.